crates/holochain_types/src/app.rs

```diff
@@ -1007,8 +1009,9 @@ pub enum PausedAppReason {
 pub enum DisabledAppReason {
     /// The app is freshly installed, and never started
     NeverStarted,
-    /// The app is partially installed, i.e. awaiting membrane proofs
-    AwaitingMemproofs,
+    /// The app is fully installed and deferred memproofs have been provided by the UI,
+    /// but the app has not been enabled. The app can be enabled via the app interface.
+    MemproofsProvided,
```

Comment:
I know it's a bit long, but
```suggestion
    NeverStartedAfterMemproofsProvided,
```

Reply:
Given our conversation today about using this state during key updates as well, the "never started" part is no longer universally applicable. I am open to other names though.

Reply:
How about `NotStartedAfterProvidingMemproofs`?